canal/config.go

```diff
@@ -113,6 +116,8 @@ func NewDefaultConfig() *Config {
 	c.Dump.ExecutionPath = "mysqldump"
 	c.Dump.DiscardErr = true
 	c.Dump.SkipMasterData = false
+	// add default value to disable mysqldump --set-gtid-purged
+	c.Dump.GtidPurged = "none"
```

Comment:
how about defining two variable for "none" and "auto"

Reply:
GtidPurged is not used that much, I think it's ok right now.

Reply:
Previously the default value was always `auto`,  will it affect other user behaviors after being changed to `none`？